cirq/ops/moment.py

```diff
@@ -81,7 +81,7 @@ def operates_on(self, qubits: Iterable[raw_types.Qid]) -> bool:
         Returns:
             Whether this moment has operations involving the qubits.
         """
-        return any(q in qubits for q in self.qubits)
+        return any(q in self.qubits for q in qubits)
```

Comment:
oh boy, how did this even work before?

Reply:
It may be faster to say
`return set(qubits) & self.qubits`
See https://stackoverflow.com/questions/2197482/efficiently-knowing-if-intersection-of-two-list-is-empty-or-not-in-python

Reply:
Done.  I don't see much difference, but it might be significant for larger numbers of qubits.